src/test/suite/fixtures/recorded/languages/css/clearUnit5.yml

```diff
@@ -0,0 +1,37 @@
+languageId: css
+command:
+  spokenForm: clear unit
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: unit}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |-
+    div {
+        padding: 25px 50px 75px 100px;
+    }
+      
+  selections:
+    - anchor: {line: 1, character: 4}
+      active: {line: 1, character: 4}
+  marks: {}
+finalState:
+  documentContents: |-
+    div {
+        padding: 25 50 75 100;
```

Comment:
I think this is nice.  wdyt @auscompgeek @AndreasArvidsson

Reply:
Do I have this correct that it will operate on all units in the value if you target the key? That is fine.

Reply:
yep!

Reply:
Seems fine, not sure what else it could do in that case anyway.